modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/util/ObjectRelationshipNameUtil.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.web.internal.util;
+
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.util.StringUtil;
+
+/**
+ * @author Rodrigo Paulino
+ */
+public class ObjectRelationshipNameUtil {
```

Comment:
@rodrigopaulino I found this name too specific, because it doesn't make sense to add more methods besides `getRelationshipName`. It could be just `ObjectRelationshipUtil`.

I would appreciate hearing your thoughts on this 😄

Reply:
The convention is to only use the name of an existing interface if its util mirrors it. Here's the [link](https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752634/API+conventions+in+Liferay#[inlineExtension]Util-classes) for the convention. :)

Reply:
I didn't know, thanks for sharing this with me @rodrigopaulino!